tests/transforms/test_tape_expand.py

```diff
@@ -66,7 +67,7 @@ def test_device_and_stopping_expansion(self):
         """Test that passing a device alongside a stopping condition ensures
         that all operations are expanded to match the devices default gate
         set"""
-        dev = qml.device("default.qubit.legacy", wires=1)
+        dev = DefaultQubitLegacy(wires=1)
```

Comment:
Why can't we just use `default.qubit` here?

Reply:
Let me look into it.

Reply:
Ah because `stopping_condition` is specific to the legacy device API. The new device API don't use that anymore.

Reply:
Ah got it. In that case I'll resolve this conversation.